test/unit/api_spec.js

```diff
@@ -979,6 +979,13 @@ describe('api', function() {
       expect(viewport.width).toEqual(1262.835);
       expect(viewport.height).toEqual(892.92);
     });
+    it('gets viewport respecting "dontFlip" argument', function () {
+      var flipped = page.getViewport(1.5, 135);
```

Comment:
Nit: Please use `let` instead of `var` in new code.

Reply:
ok

- [ ] switch to es6